cabby/geo/.gitignore

```diff
@@ -1 +1,2 @@
-__pycache__\*
+__pycache__
+__pycache__
```

Comment:
This shouldn't be necessary -- the top level gitignore already excludes pycache in all sub-directories.